boilerplate/openshift/golang-osd-operator/csv-generate/catalog-build.sh

```diff
@@ -22,10 +22,12 @@ while getopts "o:c:r:" option; do
 done
 
 # Checking parameters
-check_mandatory_params operator_channel operator_name  
+check_mandatory_params operator_channel operator_name
 
 # Parameters for the Dockerfile
 SAAS_OPERATOR_DIR="saas-${operator_name}-bundle"
+BUNDLE_DIR="${SAAS_OPERATOR_DIR}/${operator_name}"
+REGISTRY_IMG="quay.io/app-sre/${operator_name}-registry"
```

Comment:
This needs to go away. It's [set in standard.mk](https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/standard.mk#L32) now. Overriding it here breaks ability to test locally with a personal registry repo.

Reply:
```suggestion
```

Reply:
Actually, looking closer, this *shouldn't* affect anything because the var is unused in this script (and shouldn't bleed into any other scripts from here). I'll merge this and we can clean it up later.

Reply:
Fixed this in the rebase.